### PR TITLE
Fix check for Qwt>=6.2.0 for QWT_POLAR

### PR DESCRIPTION
**Pull request by MehdiChinoune**
Also set QWT_POLAR_VERSION=0x060200

## Description

It was checking a wrong variable.

This is part of https://github.com/qgis/QGIS/commit/720d22d

### CODE REVIEW

**Review by m-kuhn — APPROVED** [review]

**Comment by m-kuhn** [comment]
@stefanuhrig do you have an idea what happened to the hana test?

**Comment by MehdiChinoune** [comment]
I think the failure is due https://github.com/actions/runner-images/issues/9557

**Comment by m-kuhn** [comment]
Thanks @MehdiChinoune !

**Comment by stefanuhrig** [comment]
@m-kuhn As far as I can see, the hana test issue has been resolved. If there are any further issues, please reach out to me.

File: .ci/run_tests.sh [hunk 1]
```diff
@@ -129,7 +125,7 @@ mkdir -p /tmp/minio_tests/test-bucket && chmod -R 777 /tmp/minio_tests
 # Create an empty webdav folder with appropriate permissions so www user can write inside it
 mkdir -p /tmp/webdav_tests && chmod 777 /tmp/webdav_tests
 
-docker-compose \
+docker compose \
```
Comment (strk) [inline .ci/run_tests.sh:128]: Why this change ? Is related to QWT at all ? It broke runs with podman, see GH-57724

Comment (MehdiChinoune) [reply .ci/run_tests.sh:128]: At the time GH dropped docker compose v1 (before bring it back after users complained).
They are planning to drop it on July, 9 actions/runner-images#9692

Comment (strk) [reply .ci/run_tests.sh:128]: I don't think GH runs .ci/run_tests.sh though, that script is for humans usage

**Comment by cp4638861** [comment]
Try.
